src/components/TertiaryNav/TertiaryNav.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { Switch, Route, NavLink, RouteComponentProps } from 'react-router-dom';
+import { withRouter } from 'react-router';
+import * as styles from './TertiaryNav.sass';
+import IReactComponentProps from '../../common/structures/IReactComponentProps';
+
+class TertiaryNavBase extends React.Component<IReactComponentProps & RouteComponentProps<{}>> {
+
+	// static propTypes = {
```

Comment:
There are a couple of commented code blocks in this file that need to be taken care of.